scripts/env/pull-env.sh

```
@@ -141,14 +141,53 @@ pull_all_services() {
     xargs -P0 -I {} bash -c 'pull_env_file $@' _ {} $env
 }
 
+pull_act_secret() {
+  if [ ! -f ".env.ci" ]; then
+    cat <<EOF > .env.ci
+# Local environment variables
+# Get your infisical service token on your provided account
+INFISICAL_CI_TOKEN=''
+EOF
+    err_token_missing
+    exit
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)